qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/CassandraMetricsClientLimitMultiplier.java

```diff
@@ -49,7 +50,7 @@ public static ClientLimitMultiplier create(Supplier<QosCassandraMetricsRuntimeCo
                 installConfig.throttlingStrategy());
 
         CassandraMetricMeasurementsLoader cassandraMetricMeasurementsLoader = new CassandraMetricMeasurementsLoader(
-                () -> runtimeConfig.get().cassandraHealthMetrics(), metricsService);
+                () -> runtimeConfig.get().cassandraHealthMetrics(), metricsService, Executors.newSingleThreadScheduledExecutor());
```

Comment:
Should allow injecting this from further up so that the server framework can manage it if desired.